src/components/structures/ScrollPanel.tsx

```diff
@@ -226,7 +226,15 @@ export default class ScrollPanel extends React.Component<IProps> {
 
     private onScroll = ev => {
         // skip scroll events caused by resizing
-        if (this.props.resizeNotifier && this.props.resizeNotifier.isResizing) return;
+        if (this.props.resizeNotifier && this.props.resizeNotifier.isResizing) {
+            debuglog("skipping onScroll due to isResizing", this.getScrollNode().scrollTop);
+            return;
+        }
+        // skip scroll events which are triggered by backfilling (i.e. while loading permalinks)
+        if (this.isFilling) {
+            debuglog("skipping onScroll due to isFilling", this.getScrollNode().scrollTop);
+            return;
+        }
```

Comment:
> we're not sure if this actually fixes an issue at the moment, and needs a regression test to do so.

@ara4n thinks the problem might still be reproducible even with this fix but do these changes at least make sense?

We should merge in that case anyway